assets/styles/parts/utils/_colors.scss

```diff
@@ -6,4 +6,8 @@ All variables are available here:
 https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_colors.scss
 */
 
-$base-primary-color: #FF86D6;
\ No newline at end of file
+$base-primary-color: #FF86D6;
+
+$base-gray-color: #999999;
```

Comment:
https://infinum.github.io/eightshift-docs/docs/basics/writing-styles#never-use-the-actual-color-name-as-a-variable-name